docs/pt/docs/advanced/templates.md

```diff
@@ -0,0 +1,77 @@
+# Templates
+
+Você pode usar qualquer template engine, que você quiser com o **FastAPI**.
```

Comment:
```suggestion
Você pode usar qualquer *template engine*, que desejar com o **FastAPI**.
```

Reply:
I Highly suggest to use italic mode when using some English expression to use to be more readable.

Reply:
Another suggestion here is to join some sentences that in english have a lot of cohesion but when reading and speaking in portuguese sounds ambiguos, like `Voce pode utilizar qualquer *template engine* que desejar com **FastAPI**, como o Jinja2, o mesmo utilizado pelo Flask e outras ferramentas.`

Reply:
Now that you mention it, i can see the ambiguity

Reply:
Is there any reason why do not translate `template engine` as `gerador de templates`?

At the company a work to, we use this term all the time when handle template engine (like jinja2, as example).

I'd had the same experience talking to others developers as well